tests/paddle/test_paddle_autolog.py

```diff
@@ -0,0 +1,160 @@
+import pytest
+import paddle
+import mlflow
+import mlflow.paddle
+
+NUM_EPOCHS = 6
+
+pytestmark = pytest.mark.large
+
+
+@pytest.fixture
+def dataset():
+    train_dataset = paddle.text.datasets.UCIHousing(mode="train")
+    eval_dataset = paddle.text.datasets.UCIHousing(mode="test")
+    return train_dataset, eval_dataset
+
+
+@pytest.fixture
+def test_model():
```

Comment:
Can I push some commits to refactor the code structure a bit?

Reply:
Yes, sure!